projects/finder-frontend/docker-compose.yml

```diff
@@ -30,9 +30,17 @@ services:
       - search-api-app
       - memcached
       - nginx-proxy
+      - govuk-account-manager-prototype-app
     environment:
       ASSET_HOST: finder-frontend.dev.gov.uk
       VIRTUAL_HOST: finder-frontend.dev.gov.uk
+      FEATURE_FLAG_ACCOUNTS: "enabled"
+      GOVUK_ACCOUNT_OAUTH_CLIENT_ID: transition-checker-id
+      GOVUK_ACCOUNT_OAUTH_CLIENT_KEY: "-----BEGIN EC PRIVATE KEY-----\nMHcCAQEEID+UE4jB5QtUFOtekHJliaSB8K2xQU2AS4xczOLAqvCUoAoGCCqGSM49\nAwEHoUQDQgAEjMgE/d6Bdu1K17BTjNycDvIXKSmUgZ5YYoayU3gGqTLhgefuK2qb\no99Wx+SuvdW/GRvIlUIW1ooNRdQ3QFwwCw==\n-----END EC PRIVATE KEY-----\n"
+      GOVUK_ACCOUNT_OAUTH_CLIENT_KEY_UUID: "898d62b7-eed9-464a-a4ae-9d9e08bd9bee"
+      OIDC_CLIENT_ID: transition-checker-id
+      OIDC_CLIENT_SECRET: transition-checker-secret
```

Comment:
I've renamed these in https://github.com/alphagov/finder-frontend/pull/2214.

`OIDC_CLIENT_ID` has been removed and replaced with `GOVUK_ACCOUNT_OAUTH_CLIENT_ID`.
`OIDC_CLIENT_SECRET` is now `GOVUK_ACCOUNT_OAUTH_CLIENT_SECRET`.

Hopefully this will make the names more consistent with those that already exist.